docs/views/about.html

```diff
@@ -24,15 +24,45 @@ <h1 class="govuk-heading-xl">About</h1>
       <p class="govuk-body-l">
         The GOV.UK Prototype Kit provides a simple way to make interactive prototypes that look like pages on GOV.UK. These prototypes can be used to show ideas to people you work with, and to do user research
       </p>
-
-      <h2 class="govuk-heading-m">Community</h2>
-
-      <p>We have two Slack channels for the GOV.UK Prototype Kit. You'll need a government email address to join.</p>
+      
+      <h2 class="govuk-heading-m">Support</h2>
+      
+      <p>The GOV.UK Prototype Kit is maintained by the Government Digital Service. If you’ve got a question or need support you can:</p>
       <ul class="govuk-list govuk-list--bullet">
-        <li><a href="https://ukgovernmentdigital.slack.com/messages/prototype-kit/">Slack channel for users of the GOV.UK Prototype Kit</a></li>
-        <li><a href="https://ukgovernmentdigital.slack.com/messages/prototype-kit-dev/">Slack channel for developers of the GOV.UK Prototype Kit</a></li>
+        <li>email 
+          <a href="mailto:govuk-design-system-support@digital.cabinet-office.gov.uk" class="govuk-link" data-hsupport="email">
+            govuk-design-system-support@digital.cabinet-office.gov.uk
+          </a>
+        </li>
+        <li>
+          <a href="https://ukgovernmentdigital.slack.com/messages/govuk-prototype-kit" class="govuk-link" data-hsupport="slack">
+            get in touch on Slack
+          </a> 
+          (<a class="govuk-link" href="slack://channel?team=T04V6EBTR&amp;id=C0647LW4R" data-hsupport="slackapp">open in app</a>)
+        </li>
+        <li>
+          <a href="https://github.com/alphagov/govuk-prototype-kit/issues" class="govuk-link" data-hsupport="github">
+            view known issues on GitHub
+          </a>
+        </li>
       </ul>
 
+      <h2 class="govuk-heading-m">Contributing</h2>
+      
+      <p>If you’ve got an idea or suggestion you can:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>
+          <a href="https://ukgovernmentdigital.slack.com/messages/govuk-prototype-kit-dev" class="govuk-link" data-hsupport="slack">
+            get in touch on developer Slack channel
```

Comment:
Perhaps

```suggestion
            get in touch on the developer Slack channel
```

would make more sense?